codegen/generator/nodejs/templates/src/method_comment.ts.tmpl

```diff
@@ -1,32 +1,41 @@
 {{ define "method_comment" }}
-  {{- if .Description -}}
-    {{- $commentLines := CommentToLines .Description }}
-    {{- /* we split the comment string into a string slice of one line per element */ -}}
-    {{- $required := GetRequiredArgs .Args -}}
-    {{- $optionals := GetOptionalArgs .Args -}}
-    {{- $deprecationLines := FormatDeprecation .DeprecationReason }}
+	{{- $required := GetRequiredArgs .Args }}
+	{{- $optionals := GetOptionalArgs .Args }}
+	{{- $argsDesc := ArgsHaveDescription .Args }}
+	{{- $deprecationLines := FormatDeprecation .DeprecationReason }}
+	{{- if or .Description $argsDesc .IsDeprecated }}
+		{{- $commentLines := CommentToLines .Description }}
+		{{- /* we split the comment string into a string slice of one line per element */ -}}
 {{""}}
 {{""}}
   /**
-  {{- range $commentLines }}
+		{{- range $commentLines }}
```

Comment:
Nit: Why so many tab? 

:bulb: It's definitely not a merge blocker, we can take care of this on #4294

Reply:
So, this is just to be able to view the logic part of the template compared to the actual formatting (which needs to be 2-space indented, etc)
Otherwise, if all the logic is at the same level of indentation, it's hard to follow anything.